omero/sysadmins/version-requirements.txt

```diff
@@ -149,7 +149,7 @@ The planned changes for 5.2 depend upon our support of
   * [5.2] 3.4 dropped, 3.5 deprecated, 3.6 not supported
```

Comment:
I don't think we can really say 3.5 is deprecated if we aren't supporting 3.6. This will have to be changed to "3.5 recommended,"